pkg/injector/bootstrap.go

```diff
@@ -53,7 +50,7 @@ func (wh *mutatingWebhook) createEnvoyBootstrapConfig(name, namespace, osmNamesp
 		CipherSuites:          wh.configurator.GetMeshConfig().Spec.Sidecar.CipherSuites,
 		ECDHCurves:            wh.configurator.GetMeshConfig().Spec.Sidecar.ECDHCurves,
 	}
-	config, err := bootstrap.GenerateEnvoyConfig(configMeta, wh.configurator)
+	config, err := builder.Build()
```

Comment:
The way this line is rewritten - it carries much less semantic meaning.
Looking at it (without seeing what was replaced) - I see that a builder builds config. But all 3 of these words are so generic - it carries little meaning.
I'd propose we rename `config` to `envoyBootstrapConfig` and `builder.Build()` to `bootstrapConfig.Build()` or something of that nature.

Reply:
changed to bootstrapConfig. 

[I'll point out that go explicitly prefers short variable names when it is used in a limited scope](https://github.com/golang/go/wiki/CodeReviewComments#variable-names)

In this case, the method config, and builder both appear within a few lines of the function name `createEnvoyBootstrapConfig`, and the instantion of the builder: `builder := bootstrap.Builder{...`

I don't think it's too important for a single line to have that much context, but too quickly skim a handful of lines and understand what is happening.

Anyways, just wanted to add that as a counter point, I've changed to `bootstrapConfig` which should be a happy medium